src/sources/hydra.rs

```diff
@@ -146,14 +147,39 @@ impl Worker {
     }
 }
 
+fn intersect_from_config(intersect: &IntersectConfig) -> Point {
+    match intersect {
+        IntersectConfig::Origin => {
+            info!("intersecting origin");
+            Point::Origin
+        }
+        IntersectConfig::Tip => {
+            panic!("intersecting tip not currently supported with hydra as source")
+        }
+        IntersectConfig::Point(slot, hash_str) => {
+            info!("intersecting specific points");
+            let hash = hex::decode(hash_str).expect("valid hex hash");
+            Point::Specific(slot.clone(), hash)
+        }
+        IntersectConfig::Breadcrumbs(_) => {
```

Comment:
Not sure what these are yet

Reply:
let's remove 
```
breadcrumbs: Breadcrumbs,
```
from `Stage` definition for the time being

Reply:
👍 done, but it's not possible to drop this `IntersectConfig::Breadcrumbs(_)` case. Or at least not in any obvious way.